commons/src/main/res/values/strings.xml

```diff
@@ -290,6 +290,10 @@
     <string name="confirm_storage_access_text_sd">If you don\'t see the SD card, try this</string>
     <string name="confirm_storage_access_android_text">Please allow the app accessing the selected storage on the next screen by pressing \'Use this folder\' at the bottom.</string>
     <string name="confirm_storage_access_android_text_specific">Please allow access to \'&lt;b&gt;%s&lt;/b&gt;\' on the next screen by pressing \'&lt;b&gt;Use this folder&lt;/b&gt;\' at the bottom.</string>
+    <string name="confirm_storage_access_restricted_text">Due to system restrictions, this folder is not available.</string>
+    <string name="confirm_storage_access_restricted_text_request_access">Request access</string>
```

Comment:
I guess this "Request access" string is not needed anymore, at least not for now.